.gitmodules

```diff
@@ -5,7 +5,7 @@
 [submodule "core/packages/contracts/lib/forge-std"]
 	path = core/packages/contracts/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
-	branch = v1.4.0
+	branch = master
```

Comment:
Seems like this is unrelated to the beacon light client. Lets try to keep the PRs focused.

Reply:
It's for fixing solidity tests use `prevrandao` to replace `difficulty` which is deprecated and cause CI breaking. I'll move it to another PR.